doc/GettingStarted.md

```diff
@@ -67,6 +67,9 @@ Vitess without Docker.
 ### Install Dependencies
 
 We currently test Vitess regularly on Ubuntu 14.04 (Trusty) and Debian 8 (Jessie).
+OSX should work as well, the installation instructions are below.
```

Comment:
Could you mention the particular version of OS X that worked for you? Also include the space when referring to "OS X" in this doc.